configs/defs/cos.textproto

```diff
@@ -2913,6 +2913,7 @@ benchmark_configs: {
   compliance_note: {
     version: { cpe_uri: "cpe:/o:cos:cos_linux:93" version: "1.0.0" benchmark_document: "CIS Container-Optimized OS" }
     version: { cpe_uri: "cpe:/o:cos:cos_linux:97" version: "1.0.0" benchmark_document: "CIS Container-Optimized OS" }
+    version: { cpe_uri: "cpe:/o:centos:centos:7.*" version: "3.1.2" benchmark_document: "CIS Centos Linux 7.x" }
```

Comment:
This looks like a generic linux check and not a COS one. Can you move it into the generic_linux.textproto?

Reply:
Do you mean the whole benchmark?

Reply:
Yeah, the whole benchmark. If it's used by OSes other than COS then it should be defined in the generic linux def file.